src/prg_system_mod.F90

```diff
@@ -359,14 +362,16 @@ subroutine prg_parse_system(system,filename,extin)
        allocate(system%coordinate(3,nats))
        allocate(system%mass(nats))
        allocate(system%lattice_vector(3,3))
+       allocate(system%resname(nats))
 
        system%lattice_vector = 0.0_dp
+       system%resname = "MOL"
```

Comment:
Why do you set `resname`? Isn't it overwritten when you read the file?

Reply:
Removed